include/dfu/dfu.h

```diff
@@ -113,19 +111,19 @@ void bm_dfu_req_next_chunk(uint64_t dst_node_id, uint16_t chunk_num);
 void bm_dfu_update_end(uint64_t dst_node_id, uint8_t success, BmDfuErr err_code);
 void bm_dfu_send_heartbeat(uint64_t dst_node_id);
 
-void bm_dfu_init(BcmpDfuTxFunc bcmp_dfu_tx);
+void bm_dfu_init();
```

Comment:
Could you please specify `void` for C functions that have no parameters?
In C++ this is defined behavior,
but in C  a function with no parameters actually invokes undefined behavior (it creates a function of unknown number of arguments).

Take a look at this for a bit more in depth explanation: https://wiki.sei.cmu.edu/confluence/display/c/DCL20-C.+Explicitly+specify+void+when+a+function+accepts+no+arguments